Solutions/SeraphicSecurity/Data Connectors/SeraphicSecurityConnector.json

```diff
@@ -0,0 +1,161 @@
+      {
+          "id": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.OperationalInsights/workspaces/',parameters('workspace'),'/providers/Microsoft.SecurityInsights/dataConnectors/',guid(subscription().subscriptionId))]",
+          "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',guid(subscription().subscriptionId))]",
+          "apiVersion": "2021-03-01-preview",
+          "type": "Microsoft.OperationalInsights/workspaces/providers/dataConnectors",
+          "kind": "APIPolling",
+          "properties": {
+              "connectorUiConfig": {
+                  "id": "SeraphicWebSecurity",
+                  "title": "Seraphic Web Security",
+                  "publisher": "Seraphic",
+                  "descriptionMarkdown": "The Seraphic Web Security data connector provides the capability to ingest [Seraphic Web Security](https://seraphicsecurity.com/) events and alerts into Microsoft Sentinel.",
+                  "graphQueriesTableName": "SeraphicWebSecurity_CL",
+                  "graphQueries": [
+                      {
+                          "metricName": "Total events received",
+                          "legend": "Seraphic Web Security Events",
+                          "baseQuery": "{{graphQueriesTableName}}\n| where bd_type_s == 'Event'"
+                      },
+                      {
+                          "metricName": "Total alerts received",
+                          "legend": "Seraphic Web Security Alerts",
+                          "baseQuery": "{{graphQueriesTableName}}\n| where bd_type_s == 'Alert'"
+                      }
+                  ],
+                  "sampleQueries": [
+                      {
+                          "description": "All Seraphic Web Security events",
+                          "query": "{{graphQueriesTableName}}\n| where bd_type_s == 'Event'\n| sort by TimeGenerated desc"
+                      },
+                      {
+                          "description": "All Seraphic Web Security alerts",
+                          "query": "{{graphQueriesTableName}}\n| where bd_type_s == 'Alert'\n| sort by TimeGenerated desc"
+                      }
+                  ],
+                  "dataTypes": [
+                      {
+                          "name": "{{graphQueriesTableName}}",
+                          "lastDataReceivedQuery": "{{graphQueriesTableName}}\n| summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+                      }
+                  ],
+                  "connectivityCriterias": [
+                      {
+                          "type": "SentinelKindsV2",
+                          "value": [
+                              "APIPolling"
+                          ]
+                      }
+                  ],
+                  "availability": {
+                      "status": 1,
+                      "isPreview": true
```

Comment:
data connector "isPreview" is true, you want to keep this data connector in preview?

Reply:
fixed